tests/mime/mime-dec-parse-full-msg-test01/test.yaml

```diff
@@ -0,0 +1,48 @@
+# *** Add configuration here ***
```

Comment:
nit: These are indicators to add more conf like min version required and should/can be removed when no additional conf is needed

Reply:
ah. i see its done in the next commit. I think these fixes should be a part of this commit.